src/configs/redoc/text-api/spec.yml

```diff
@@ -0,0 +1,330 @@
+openapi: 3.0.0
```

Comment:
@wfabjanczuk, here I pasted the content from the Customer Accounts API as it was easiest for me to check if everything works by doing it this way. Feel free to change it for the new one.